server/controllers/auth.js

```diff
@@ -109,7 +109,9 @@ exports.loadUser = asyncHandler(async (req, res, next) => {
         name: user.name,
         email: user.email
       },
-      profile
+      profile: {
+        photo: profile.photo
```

Comment:
Why don't we return the entire profile here rather than just the `photo` field? (We will need to use the logged in user's profile in various components across the app, excluding it here would mean making another API call to separately fetch the profile)